libraries/lib-datahandler/src/main/thrift/sw360.thrift

```diff
@@ -12,6 +12,11 @@
 namespace java org.eclipse.sw360.datahandler.thrift
 namespace php sw360.thrift
 
+enum Quadratic {
```

Comment:
Can we re-use the existing `Ternary` enum? or can you me  let's know the specific need for this new `Quadratic` enum?

Reply:
`Ternary` got 3 values while we only need 2 values for theses fields. So, we decided to create new enum.